### PR TITLE
New Perforce Command Line Client cheat sheet

### PR DESCRIPTION
**Pull request by lizlam**
https://duck.co/ia/view/perforce_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [Perforce](https://duck.co/ia/view/perforce)

**Description**: Perforce cheat sheet for the command line client.

**Example Query**: [perforce p4](https://beta.duckduckgo.com/?q=perforce p4), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**: http://www.perforce.com

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/perforce)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/perforce)*

**Comment by daxtheduck** [comment]
@lizlam Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub username:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by jophab** [comment]
@lizlam there are two files in this pr.
Lest remove the one for perforce worlkshop

**Comment by lizlam** [comment]
@jophab Oh sorry about that, seems like that's a side effect of creating a new branch before previous pull requests are fully merged. I'll remove it now.

File: share/goodie/cheat_sheets/json/perforce.json [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+
+    "template_type": "terminal",
+
+    "section_order": [
+        "Develop",
+        "Local Security",
+        "Server Security",
+        "Investigate",
+        "Labels"
+
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/perforce.json]: Lose the blank line.

**Comment by moollaza** [comment]
@lizlam is this intended to be different from Perforce Workshop Cheat Sheet? If so, you need to remove the `perforce` alias from that Cheat Sheet's JSON, because right now that alias causes the query "perforce cheat sheet" to trigger that IA, meaning this Cheat Sheet won't work.

The triggers and aliases need to be unique across all cheat sheets :+1:

**Comment by moollaza** [comment]
@lizlam I went ahead an made a PR. This one seems ready for merge :+1: 

https://github.com/duckduckgo/zeroclickinfo-goodies/pull/2275

**Comment by moollaza** [comment]
@lizlam uh oh -- because of your commit history, this PR now conflicts with master. I can manually pull in your PR's. In the future, try to ensure you use 1 branch per IA so your commits are only adding what’s needed for the given PR

**Comment by lizlam** [comment]
@moollaza Thank you for starting the new PR.  I will definitely make to use new branches for every IA in the future.  Thanks!

**Pull request closed without merging**